ui/admin/tests/acceptance/aliases/selectors.js

```diff
@@ -5,3 +5,8 @@
 
 export const FIELD_DESTINATION_ID = '[name="destination_id"]';
 export const FIELD_HOST_ID = '[name="authorize_session_arguments"]';
+
+export const CLEAR_DROPDOWN_SELECTOR =
```

Comment:
I do believe, none if these selectors are need since they already exists within `commonSelectors`.

- DROPDOWN_BUTTON_SELECTOR = [you can use this](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/helpers/selectors.js#L20).
- CLEAR_DROPDOWN_SELECTOR = [you can use this](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/tests/helpers/selectors.js#L22)

Reply:
Fixed